notebook/notebookapp.py

```diff
@@ -870,13 +868,17 @@ def init_webapp(self):
             ssl_options['keyfile'] = self.keyfile
         if self.client_ca:
```

Comment:
Is there a reason this entire if statement isn't moved down below in the else block?